conda-forge.yml

```diff
@@ -1,6 +1,8 @@
+bot:
+  inspection: update-grayskull
```

Comment:
Uses Grayskull for version updates to refresh dependencies as noted in issue: https://github.com/conda-forge/pynvml-feedstock/issues/16